website/src/components/explorer/details.jsx

```diff
@@ -0,0 +1,187 @@
+// Contains code adapted from https://observablehq.com/@nrabinowitz/h3-index-inspector under the ISC license
+
+import React, { useCallback } from "react";
+import {
+  cellToBoundary,
+  cellToParent,
+  getResolution,
+  cellToChildren,
+  gridDisk,
+  getBaseCellNumber,
+  isPentagon,
+  getIcosahedronFaces,
+  cellToLatLng,
+  h3IndexToSplitLong,
+  cellArea,
+  edgeLength,
+  originToDirectedEdges,
+  UNITS,
+} from "h3-js";
+
+function h3IndexToDigits(h) {
+  const THREE_BITS = 0x1 | 0x2 | 0x4;
```

Comment:
Super minor nit: Since JS is interpreted, these operations would be executed on each function call. Maybe just replace this with:

```suggestion
  const THREE_BITS = 7;
```